testing/test_config.py

```diff
@@ -1035,8 +1072,8 @@ def distributions():
 
     monkeypatch.setenv("PYTEST_DISABLE_PLUGIN_AUTOLOAD", "1")
     monkeypatch.setattr(importlib_metadata, "distributions", distributions)
-    monkeypatch.setitem(sys.modules, "mytestplugin", PseudoPlugin())
-    config = testdir.parseconfig(*parse_args)
+    monkeypatch.setitem(sys.modules, "mytestplugin", PseudoPlugin())  # type: ignore[misc]
```

Comment:
wasn't sure how to workaround mypy expecting `type.ModuleType` arg here.

Reply:
In this situation you can also do

```py
from types import ModuleType
from typing import cast

mytestplugin = cast(ModuleType, PseudoPlugin()
monkeypatch.setitem(sys.modules, "mytestplugin", mytestplugin)
```

but the type-ignore is also fine, however you prefer.